### ISSUE TITLE
[Issue]: Setup - Failure to install Package in Docker [Ubuntu 22.04]

### ISSUE THREAD

**Issue by kiritigowda** [ROCm/rocDecode#308]
### Problem Description

Due to the issue below, fails to install - `libstdc++-12-dev`

## Inside U22 - Docker
```
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> print(platform.version())
#111~20.04.1-Ubuntu SMP Mon Mar 11 15:44:43 UTC 2024
>>> print(platform.version())
#111~20.04.1-Ubuntu SMP Mon Mar 11 15:44:43 UTC 2024
>>> print(platform.version())
#111~20.04.1-Ubuntu SMP Mon Mar 11 15:44:43 UTC 2024
>>> exit()
cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.4 LTS"
root@simon:~/build# cat /etc/os-release  
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```
## U22 Baremetal

```
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>  import platform
  File "<stdin>", line 1
    import platform
IndentationError: unexpected indent
>>> import platform
>>> print(platform.version())
#26~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue Mar 12 10:22:43 UTC 2
cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

### Operating System

Ubuntu 20/22

### CPU

ALL

### GPU

AMD Radeon Pro W6800, AMD Radeon VII

### ROCm Version

ROCm 6.0.0

**Comment by kiritigowda** [comment]
## Ubuntu 22 Docker

platform.version() return 20.04
```
root@simon:~# cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
root@simon:~# python3
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platfrom
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'platfrom'
>>> import platform
>>> print(platform.version())
#115~20.04.1-Ubuntu SMP Mon Apr 15 17:33:04 UTC 2024
>>> 

```

**Comment by kiritigowda** [comment]
Fixed with #331
